mealie/core/config.py

```diff
@@ -151,6 +151,11 @@ def public_db_url(cls, v: Optional[str], values: dict[str, Any]) -> str:
     DEFAULT_EMAIL: str = "changeme@email.com"
     DEFAULT_PASSWORD: str = "MyPassword"
 
+    USE_LDAP_AUTH: bool = False
```

Comment:
Can you change `USE_LDAP_AUTH` to `LDAP_AUTH_ENABLED` to keep the prefix consistent with the other variables?